### ISSUE TITLE
Docs on how to customize Scribus 

### ISSUE THREAD

**Issue by luzpaz** [nitramr/scribus-indigo#40]
@nitramr thanks for all the awesome work on Scribus-Indigo so far. I've opened a ticket on Mantis mentioning you in https://bugs.scribus.net/view.php?id=14498 this is a ticket requesting README.md needed for the Scribus Iconsets directory.

> **Icons:** https://github.com/scribusproject/scribus/tree/master/resources/iconsets
> Martin at Scribus-Indigo has made some progress in this area. Eventually we can 
> have it that people would easily be able to customize Scribus to their whims. 
> **We need to document how they can do that**.
> 

Can we start to document how users could customize Scribus in a step-by-step fashion?

**Comment by nitramr** [comment]
Something like that?
https://github.com/nitramr/scribus-indigo/wiki/Customize-Scribus

**Comment by luzpaz** [comment]
Can we use SVGs for icons as well ? They are more HiDPI friendly, right ?

**Comment by nitramr** [comment]
Good idea. To achieve it we can modifiy the iconmanager.cpp and load SVG's via:

http://doc.qt.io/qt-5/qsvgrenderer.html

```
QSvgRenderer svgRenderer( somesvgfile );
QPixmap pix( svgRenderer.defaultSize() ); // scale factor has to be applied
QPainter pixPainter( &pix );
svgRenderer.render( &pixPainter );
```
Based on this 'pix' Pixmap we can generate new QIcons with enabled / disabled states (as I used it for my themes). Based on screen type we can set a scale factor by generating the new QPixmaps.

I will make some tests to check it out.

But one hint at this point. We have to check if we can use the same svg file for 16px* icons and 32px* icons because of level of details. Maybe we still would need two different versions. (16px* and grater than 16px*).

*) pixel sizes demonstrate the relative scaling between icons.

**Comment by nitramr** [comment]
It is simplier as expected. Qpixmap can load nativly a SVG file. I have made a test with pages panel icon here: https://github.com/nitramr/scribus-indigo/commit/a8789628d03f1586202a917d6878a12ed1bba719

Bad thing of using SVG files is the file size. Regular png icon need ca 300 byte, SVG icon need 3,5KB
In latest commit there is still a scale factor missing to render the icon in right size.

My idea would be to set a normal render size by loading of any icon. Based on screen type it can by multiplied by 2.

@luzpaz do we have svg files for all used Scribus icons? If yes I can make some more tests for icons in different sizes (16px, 22px, 32px etc.)

**Comment by luzpaz** [comment]
opened a ticket at https://bugs.scribus.net/view.php?id=14528
